src/main/scala/org/opennetworkinsight/dns/DNSPostLDA.scala

```diff
@@ -24,15 +23,10 @@ object DNSPostLDA {
                  sqlContext: SQLContext,
                  logger: Logger) = {
 
-
     logger.info("DNS post LDA starts")
 
-    val topics = sc.broadcast(ipToTopicMixes)
-    val words = sc.broadcast(wordToProbPerTopic)
-
-
     val totalDataDF = {
-      sqlContext.parquetFile(inputPath.split(",")(0))
+      sqlContext.read.parquet(inputPath.split(",")(0))
```

Comment:
I am having serious doubts about the role of CSVs and TSVs in the whole design... we're seeing fields that contain commas and that can mess things up.... 

it seems like parquet is handling them okay so far but I'd like to be really sure that there are no problems with  this going forward